Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -412,7 +412,12 @@ void MarlinUI::draw_status_screen() {
 
         #if ENABLED(SHOW_REMAINING_TIME)
           if (!(ev & 0x3)) {
-            duration_t estimation = elapsed.value * (100 * (PROGRESS_SCALE) - progress) / progress;
+            duration_t estimation =
+              #if ENABLED(USE_M73_REMAINING_TIME)
+                get_remaining_time();
```

Comment:
Would there be value such that it returns the old, calculated way if set_remaining_time was never called? I'm thinking about people that use different slicers. The UI would continue to provide an estimate regardless of the slicer inserting the value or not.

Reply:
You are right, this could be a case for someone. I think, it should be hard to fix it.